src/robusta/core/sinks/robusta/robusta_sink.py

```
@@ -120,6 +120,13 @@ def _on_scan_updated(
 
         self.dal.set_scan_state(scan_id, state, metadata)
 
+    def _on_config_init(self) -> None:
+        # make sure cluster status and periodic check start after all config has been reloaded successfuly.
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)